snow/consensus/snowstorm/conflicts/tx.go

```diff
@@ -19,8 +19,8 @@ type Tx interface {
 	// Epoch this transaction was issued in.
 	Epoch() uint32
 
-	// Restrictions returns a list of transition IDs that need to be performed
-	// in this transaction's epoch or later if this transaction is accepted.
+	// Restrictions returns a list of transition IDs such that this transaction can't
+	// be accepted if the transition occurs in an epoch earlier than this transaction's epoch.
```

Comment:
I think this is clearer in the other direction just because of the word restrictions, since it's actively restricting other transactions.
```// Restrictions returns a list of transition IDs that cannot be performed prior to this transaction's epoch if this transaction is accepted.```

Reply:
`// Restrictions returns a list of transition IDs that cannot be performed prior to this transaction's epoch if this transaction is accepted.`

My concern is that this doesn't unambiguously answer the question, "what if the transaction is not accepted? Then can the these transitions occur before the transaction's epoch?"

Reply:
I mean if a transaction is not accepted then it doesn't impact anything.